package.xml

```diff
@@ -22,11 +22,13 @@
   <depend>fuse_publishers</depend>
   <depend>fuse_variables</depend>
   <depend>geometry_msgs</depend>
+  <depend>nav_msgs</depend>
   <depend>pluginlib</depend>
   <depend>roscpp</depend>
+  <depend>sensor_msgs</depend>
   <depend>std_srvs</depend>
-  <depend>tf2</depend>
   <depend>tf2_2d</depend>
+  <depend>tf2</depend>
```

Comment:
Can you "unmove" tf2?

Reply:
Done. I remove the commit where I ran `!sort -u` in vim.